content/reference/current/installation/introduction/index.md

```diff
@@ -27,14 +27,16 @@ Gatling Enterprise consists of:
   * Continuous Integration plugins for Jenkins, Bamboo and TeamCity or any solution that can trigger a bash script
 - Gatling injectors to be used like standard Gatling OSS, with extra features so Gatling Enterprise can collect data from them
 
-## Document Conventions
+## Documentation Conventions
 
-In this document, you'll find several mentions to some placeholders in capital letters.
+In this documentation, you'll find several mentions to some placeholders in capital letters.
 
-- REPLACE_WITH_YOUR_REPOSITORY_URL: the URL of the private repository that you were given alongside with your license key
+### Repository URL
```

Comment:
**question(out of scope for this PR):** Can we find another name? Because repository is used for two different things:
 * git repository
 * download platform